lib/config.js

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+const bedrock = require('bedrock');
+const config = bedrock.config;
+require('bedrock-jobs');
+
+// permissions
+const permissions = config.permission.permissions;
+permissions.DOCUMENT_SERVER_ACCESS = {
+  id: 'DOCUMENT_SERVER_ACCESS',
+  label: 'Access Documents',
+  comment: 'Required to access documents.'
+};
+permissions.DOCUMENT_SERVER_CREATE = {
```

Comment:
Should this be `DOCUMENT_CREATE`? Is this for a capability to create a document or a server?